app/models/concerns/orchestration/dns.rb

```diff
@@ -55,7 +55,12 @@ def rebuild_dns
   end
 
   def queue_dns
-    return unless (dns? || dns6? || reverse_dns? || reverse_dns6?) && errors.empty?
+    return unless dns? || dns6? || reverse_dns? || reverse_dns6?
+    unless errors.empty?
+      logger.warn("Not queueing DNS: #{errors.full_messages.to_sentence}")
```

Comment:
I think, it's unclear if the errors are on the host object or nic just by reading the logs:
How about something like this (untested):
```ruby
logger.warn("Not queueing DNS because of Nic (#{identifier}/#{mac}) errors: #{errors.full_messages.to_sentence}") if errors.any?
logger.warn("Not queueing DNS because of Host errors: #{host.errors.full_messages.to_sentence}") if host.errors.any?
```

Same for DHCP and TFTP.